Src/Plugins/AzureDevOpsConfiguration/AZC101.AzureDevOpsConfiguration.json

```diff
@@ -0,0 +1,38 @@
+{
+  "SharedStringsFileName": "AzureDevOpsConfiguration.SharedStrings.txt",
+  "ValidatorsAssemblyName": "AzureDevOpsConfiguration.dll",
+  "Definitions": [
+    {
+      "Id": "AZC101",
+      "Name": "BuildDefinitionShouldBeSecure",
+      "Level": "Error",
+      "Description": "Make sure build definitions in AzureDevOps are secure",
+      "FileNameAllowRegex": "$BuildDefinitionFiles",
+      "MatchExpressions": [
+        {
+          "Id": "AZC101/240",
+          "Name": "BuildDefinitionShouldBeSecure/DontMakeSecretsAvailableToForkedBuilds",
+          "Message": "For GitHub public repositories, it is possible that people from outside the organization can create forks and run builds on the forked repo. In such a case, if this setting is wrongly left enabled, outsiders can get access to build pipeline secrets that were meant to be internal.\r\nRecommendation: 1. Navigate to the build pipeline using resource link -->  2. Triggers --> 3. Pull request validation --> 4. Forks --> 5. Uncheck 'Make secrets available to builds of forks'.",
```

Comment:
>Message [](http://example.com/codeflow?start=11&length=7)

These messages aren't acceptable, they are entirely static. The message should report something around the identity of the build definition.

Reply:
pls review the latest update about message